spec/requests/health_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Health Controller', type: :request do
+  describe 'GET #show' do
+    context 'when the application is up' do
+      it 'returns a successful response with HTML' do
+        get '/'
```

Comment:
Always add aline break between each block in the Arrange, Act and Assert (AAA) Pattern:

```suggestion
        get '/'
        
```